brave/browser-laptop
# Removes all non-primitive types from BookmarksToolbar and BookmarkTool…

…barButton

Submitter Checklist:


Resolves #9712

Auditors: @bsclifton

Test plan:

**Note**: Reordering in context menu is not working. Will create issue for it.

1)
- right click on a blank bookmark toolbar
- context menu should be displayed

2)
- right click on a bookmark
- context menu should be displayed

3)
- right click on a bookmark folder
- context menu should be displayed

4)
- left click on a bookmark folder
- context menu should be displayed with items in the folder

5)
- create so many bookmark items that you get overflow menu
- check that overflow menu is correctly displayed 

6)
- check if bookmarks are displayed and updated correctly

7) DND test
- try to reorder bookmarks on the bookmark toolbar
- try to add bookmark into a folder
- try to bookmark a tab with dragging lock icon on the bookmark toolbar


Reviewer Checklist:

Tests
